ubyssey/templates/about/index.html

```diff
@@ -0,0 +1,156 @@
+		</p>
+		<p class="o-paragraph">
+		Joining The Ubyssey is a wonderful way to get involved on campus. The newsroom always knows what’s happening at UBC, sometimes before it happens. It is also a place to meet interesting and motivated people; many long-term friendships have started in the newsroom. Stop by room 2208 in the new SUB any time and remember: you come in with no experience, but you won’t leave that way.
+		</p>
+		<p class="o-paragraph">
+		Students have done the journalism thing at UBC since 1918. For almost as long as there has been a UBC, there has been a UBC student newspaper. The Ubyssey got its start in 1918, supported by a $2 student fee. Considering inflation rates over the past century, The Ubyssey, with its current fee a mere $6, is doing quite well.
+		</p>
+		<p class="o-paragraph">
+		The Ubyssey has played host to a wide array of budding writers. Famed poet Earle Birney edited the paper in the 1920s. Canadian icon Pierre Berton held the reins during World War II and freely admits that he came to UBC to skip his classes and work on the paper. Future Prime Minister John “Chick” Turner was an associate sports editor. Following World War II, some of the finest journalists to grace our country began their careers here, including Allan Fotheringham, Peter Worthington and Michael Valpy.
+		</p>
+	</div>
+
+
+    <div class="cont icon-cont-row">
+
+      <div class="col slide-left">
+        <h2 class="o-heading o-heading--blue">Our alumni</h2>
+        <p>
+        Today, you can see our alumni in newspapers across the country. Vaughn Palmer and Katherine Monk of the Vancouver Sun got their start here, as did sports pundit Tony Gallagher of The Province. Among our more recent alumni are Bruce Arthur of The National Post, and Jon Woodward and Sarah Galashan of CTV News.
+        </p>
+        <p style="font-style: italic;">
+        Who knows? You could be next…
+        </p>
+      </div>
+
+      <div class="col icon-div slide-right">
+        <img src="{% static 'images/about/alum-icon.png' %}" alt="graduation cap icon">
+      </div>
+
+    </div>
+
+    
+
+</div>
+
+
+	<hr>
+
+	<div class="c-section cont" id="acknowledgement">
+		<p class="o-paragraph">
+		The Ubyssey would like to acknowledge that our office resides and our production takes place on the ancestral, unceded territories of the xʷməθkʷəy̓əm (Musqueam), sḵwx̱wú7mesh (Squamish) and sel̓íl̓witulh (Tsleil-Waututh) nations. We make this acknowledgement to pay respect to the keepers of the land, and to help fulfill our responsibility to view our own colonial history and their present-day implications through a critical lens.
+		</p>
+	</div>
+	
+
+</div>
+
+
+{% include 'about/contact.html' %}
+
+{% block content %}
+{% endblock %}
+
+{% block footer %}
+
+{% endblock %}
+
+<script type="text/javascript">
```

Comment:
Similar to favicons, we can abstract this as templates/components/google_analytics.html that all pages can import with {% include 'components/google_analytics.html' %}